eng/pipelines/spec-gen-sdk.yml

```diff
@@ -16,9 +14,20 @@ parameters:
     type: string
     default: 'specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml'
     displayName: 'Path to API specification file (TypeSpec/OpenAPI)'
+  - name: ApiVersion
+    type: string
+    displayName: 'API version'
+  - name: SdkReleaseType
+    type: string
+    values:
+      - 'beta'
+      - 'stable'
+    default: 'beta'
+    displayName: 'SDK release type'
   - name: SkipPullRequestCreation
     type: boolean
     default: false
```

Comment:
Is this the right default? I kind of think we shouldn't do a pull request by default.

Reply:
It should have a default value otherwise pull request trigger would fail. Downstream language side would reply on 'api-version' to determine if this parameter needs to be respected or not. So, setting it with 'beta' won't matter.

Reply:
I was asking about the SkipPullRequestCreation default and wondering if it should be flipped.